--- ubertooth-2020-12-R1/host/python/specan_ui/CMakeLists.txt.orig	2020-12-25 22:40:39.000000000 +0100
+++ ubertooth-2020-12-R1/host/python/specan_ui/CMakeLists.txt	2022-07-05 17:59:34.835402785 +0200
@@ -31,7 +31,7 @@ add_custom_command(OUTPUT ${OUTPUT}/time
                    DEPENDS ${DEPS})
 
 add_custom_target(specan ALL DEPENDS ${OUTPUT}/timestamp)
-install(CODE "execute_process(COMMAND ${Python3_EXECUTABLE} ${SETUP_PY} build -b ${OUTPUT} install)")
+install(CODE "execute_process(COMMAND ${Python3_EXECUTABLE} ${SETUP_PY} build -b ${OUTPUT} install --prefix=${CMAKE_INSTALL_PREFIX} --root=\$ENV{DESTDIR})")
 
 add_custom_target(ubertooth-specan-ui ALL DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/ubertooth-specan-ui)
 install(PROGRAMS ubertooth-specan-ui DESTINATION ${INSTALL_DEFAULT_BINDIR})
--- ubertooth-2020-12-R1/host/python/ubtbr/CMakeLists.txt.orig	2020-12-25 22:40:39.000000000 +0100
+++ ubertooth-2020-12-R1/host/python/ubtbr/CMakeLists.txt	2022-07-05 18:09:55.498707030 +0200
@@ -14,7 +14,7 @@ add_custom_command(OUTPUT ${OUTPUT}/time
                    DEPENDS ${DEPS})
 
 add_custom_target(ubtbr ALL DEPENDS ${OUTPUT}/timestamp)
-install(CODE "execute_process(COMMAND ${PYTHON3} ${SETUP_PY} build -b ${OUTPUT} install -f --prefix=${CMAKE_INSTALL_PREFIX})")
+install(CODE "execute_process(COMMAND ${PYTHON3} ${SETUP_PY} build -b ${OUTPUT} install -f --prefix=${CMAKE_INSTALL_PREFIX} --root=\$ENV{DESTDIR})")
 
 add_custom_target(ubertooth-btbr ALL DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/ubertooth-btbr)
 install(PROGRAMS ubertooth-btbr DESTINATION ${INSTALL_DEFAULT_BINDIR})
